desktop/extensions-sdk/tutorials/initialize.md

```diff
@@ -8,7 +8,7 @@ Learn how to create a new Docker extension.
 
 ## Prerequisites
 
-- [Docker Desktop build with Extensions capabilities](https://github.com/docker/extensions-sdk/releases/)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 - [Docker Extensions CLI](https://github.com/docker/extensions-sdk/releases/)
 - [NodeJS](https://nodejs.org)
 - [Go](https://go.dev/dl/)
```

Comment:
@gtardif This won't be required anymore since version `v0.2.6` of the CLI.

Reply:
Yes, I saw this, wanted to avoid mixing dependencies, so I'll do a separate PR for it